spec/requests/graphql/query/batch_imports_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Retrieving Batch Imports', type: :request do
+  context "when logged in user is administrator" do
```

Comment:
It might be useful to copy some of my batch export tests for pagination tests and for non-admin user tests, to make sure the right results are returned.